### DISCUSSION TITLE
This field's default value can currently not be parsed: `uuid_generate_v4()`

### DISCUSSION THREAD

**Discussion by ziimakc** [prisma/prisma#4362] [Q&A] [answered]
After introspection i can see this comment int schema, is it ok or this should be fixed somehow?

```prisma
model Log {
  /// This field's default value can currently not be parsed: `uuid_generate_v4()`.
  uuid           String    @id @default(dbgenerated())

  @@map("log")
}
```

**Comment by ryands17** [comment, marked as answer]
Hey @ZiiMakc 👋 
That's totally fine. Prisma uses it's own `uuid` [mechanism](https://www.prisma.io/docs/reference/database-reference/database-features#prisma-features) and not what the DB gives by default. 

In your case, the value has been set by the DB and you are introspecting it and which is why we have added a comment here. This will not change the way you work with the field so there's no issue there. 

We should change the wording though here as it seems like the field is not supported. It will work totally fine though and there's an issue [here](https://github.com/prisma/prisma/issues/4369) for the same :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
